opencga-app/app/scripts/docker/opencga-java-base/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# up version?
+FROM ubuntu:16.04

```

Comment:
The webhosts are 18.04-LTS so probably makes sense to bump this up a version for consistency